.travis.yml

```diff
@@ -1,21 +1,57 @@
 language: python
 dist: xenial
 sudo: false
-python:
-  - "2.7"
+addons:
+  apt:
+    sources:
+      - google-chrome
+    packages:
+      - google-chrome-stable
+services:
+  - xvfb      
+matrix:
+  include:
+    - python: "2.7"
+      env: PLONE_VERSION=4.3.x
+    - python: "2.7"
+      env: PLONE_VERSION=5.0.x
+    - python: "2.7"
+      env: PLONE_VERSION=5.1.x
+    # - python: "2.7"
+    #   env: PLONE_VERSION=5.2.x
+    # - python: "3.7"
+    #   env: PLONE_VERSION=5.2.x
+    #   dist: xenial
+    #   sudo: true
 cache:
+  pip: true
   directories:
-    - eggs
-env:
-  - PLONE_VERSION=4.3
-  - PLONE_VERSION=5.0
-  - PLONE_VERSION=5.1
+  - eggs
+  - downloads
+  - $HOME/buildout-cache
+before_install:
+# install chrome webdriver
+  - mkdir webdriver;
+    wget https://github.com/mozilla/geckodriver/releases/download/v0.20.0/geckodriver-v0.20.0-linux64.tar.gz;
+    tar -xzf geckodriver-v0.20.0-linux64.tar.gz -C webdriver;
+    wget https://chromedriver.storage.googleapis.com/2.40/chromedriver_linux64.zip;
+    unzip chromedriver_linux64.zip -d webdriver;
+    export PATH=$PATH:$(pwd)/webdriver;
+  - mkdir -p $HOME/{eggs,downloads}
+  - ./bootstrap.sh test-$PLONE_VERSION.cfg
 install:
-  - pip install -r requirements.txt
-  - python --version
-  - sed -ie "s#test-4.x.cfg#test-$PLONE_VERSION.x.cfg#" travis.cfg
-  - mkdir -p buildout-cache/downloads
-  - buildout -c travis.cfg -N -q -t 3
-script: bin/test
-services:
-  - xvfb
+  - bin/buildout -N -t 20 -c test-$PLONE_VERSION.cfg
+  - pip install zest.pocompile
+  - pocompile src
+before_script:
+  - 'export DISPLAY=:99.0'
```

Comment:
I believe this is not necessary: https://docs.travis-ci.com/user/gui-and-headless-browsers/#using-xvfb-to-run-tests-that-require-a-gui

Reply:
removed